### PR TITLE
fix: everything

### PR DESCRIPTION
**Pull request by agilesteel**
Hi, I took the liberty of making it work again. Do you mind publishing it? Thanks!

### CODE REVIEW

File: src/main/scala/com/mayreh/thankyou/SbtPlugin.scala [hunk 1]
```diff
@@ -19,7 +19,7 @@ object SbtPlugin extends AutoPlugin {
 
   override def projectSettings: Seq[Setting[_]] = Seq(
     thankYouStars := {
-      val updateReport = update.value
+      val updateReport = updateFull.value
```
Comment (ocadaruma) [inline src/main/scala/com/mayreh/thankyou/SbtPlugin.scala:22]: Hi, could you elaborate a bit why this change is necessary?
Also I suppose this doesn't work on sbt 0.13 so we should drop 0.13 support from now on right?

**Review by ocadaruma — COMMENTED** [review]
Hi, thank you for the patch!
Left a question.

File: src/main/scala/com/mayreh/thankyou/SbtPlugin.scala (hunk 1 above)
Comment (agilesteel) [reply src/main/scala/com/mayreh/thankyou/SbtPlugin.scala:22]: The `update` task is used to collect `.callers` that was empty. I searched for it and found https://github.com/sbt/sbt/issues/4706#issuecomment-493680547 which led me to try `updateFull` instead and it worked. 🤷🏼‍♂️

Comment (ocadaruma) [reply src/main/scala/com/mayreh/thankyou/SbtPlugin.scala:22]: I see, got it.

Would you mind also update here to include only `1.6.2` ? (https://github.com/ocadaruma/sbt-thank-you-stars/blob/master/build.sbt#L11)

Comment (agilesteel) [reply src/main/scala/com/mayreh/thankyou/SbtPlugin.scala:22]: done

File: README.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ Give your dependencies stars on GitHub!
 
 This is a sbt-ported version of [thank-you-stars.](https://github.com/teppeis/thank-you-stars)
 
-## Requirement
+## Requirements
 
-sbt 0.13.x / 1.0.x
+sbt 1.4.x (tested only on 1.6.2)
```
Comment (ocadaruma) [inline README.md]: [nits] More precisely, `sbt 1.4.x or higher` ?

**Review by ocadaruma — COMMENTED** [review]
Thanks, left minor comment but almost LGTM.

File: README.md (hunk 2 above)
Comment (agilesteel) [reply README.md]: Sorry, my bad. Fixed :)

**Review by ocadaruma — APPROVED** [review]
Thanks, LGTM!

**Comment by agilesteel** [comment]
Awesome, let me know when it's published. I'm about to record a video about it right now!

**Comment by ocadaruma** [comment]
Has been released as `0.2`.
It will be visible on Maven in few hours.

**Comment by agilesteel** [comment]
It's been 15h and it's still not on maven https://search.maven.org/artifact/com.mayreh/sbt-thank-you-stars I think you need to release it on nexus or sth. I don't remember. It's been forever since I published anything 🙈

**Comment by ocadaruma** [comment]
Ah yeah seems I mistakenly executed only `sonatypeClose` instead of `sonatypeRelease`.
I released it now so I think it will be available soon.

**Comment by ocadaruma** [comment]
visible on https://repo1.maven.org/maven2/com/mayreh/sbt-thank-you-stars_2.12_1.0/0.2/ at least

**Comment by agilesteel** [comment]
Yep it's live. Awesome! My video is going to come out next Saturday. Would be cool if we could include a link to it in the readme! No pressure though 😉
